manifests/plugin/python.pp

```diff
@@ -11,12 +11,6 @@
 
   validate_hash($config)
 
-  if $::osfamily == 'Redhat' {
-    package { 'collectd-python':
-      ensure => $ensure,
-    }
-  }
```

Comment:
It's removed from 5 and 6 as well.

Source:
https://dl.fedoraproject.org/pub/epel/5/x86_64/
https://dl.fedoraproject.org/pub/epel/6/x86_64/
https://dl.fedoraproject.org/pub/epel/7/x86_64/